### PR TITLE
B707 fix dm pixarfilms

### PR DESCRIPTION
**Pull request by moodymudskipper**
Closes https://github.com/cynkra/dm/issues/707

### CODE REVIEW

**Comment by moodymudskipper** [comment]
need to test on windows machine to understand what's going on

**Review by krlmlr — COMMENTED** [review]
Thanks. We can keep both versions with an option to the function? We could use this as an example for an inconsistent dm, perhaps with auto-repair when we have it?

`check = TRUE` slows down the creation, perhaps let's add a test?

**Comment by krlmlr** [comment]
Let's not spend too much time on it. Getting rid of the duckdb failure seems more urgent.

File: R/pixarfilms.R [hunk 1]
```diff
@@ -44,7 +49,6 @@ dm_pixarfilms <- function(color = TRUE) {
   dm <-
     dm %>%
     dm_add_pk(pixar_films, film) %>%
-    dm_add_pk(pixar_people, c(film, role_type)) %>%
```
Comment (krlmlr) [inline R/pixarfilms.R:47]: Did you mean to remove this primary key?

Comment (moodymudskipper) [reply R/pixarfilms.R:47]: Yes, this is not unique and shouldn't be, you can have several screenwriters for a given movie for instance.

**Review by krlmlr — APPROVED** [review]

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**